### PR TITLE
Add support for travis-ci.

### LINKED ISSUE #479: [Proposal] Add DevOps CI service
It would be very helpful, if we could automate the ESLint and build check via a CI (Continuous Integration) service like [`Travis`](https://travis-ci.org/). It can help check the PR before merge.

### PR DESCRIPTION
**Pull request by JimiC**
***Fixes #479***

@robertohuertasm I've noticed that `Travis-CI` is already set and awaiting. If `Travis` doesn't pick up you may need to `Activate` the repository (it has a related button).

### CODE REVIEW

**Comment by JimiC** [comment]
Oooooooo. It works!

**Comment by JimiC** [comment]
Yeah \o/.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js: "6"
+script: npm run build
```
Comment (robertohuertasm) [inline .travis.yml]: Just pointing out the newline warning.
Great work @JimiC !

Comment (JimiC) [inline .travis.yml]: Cause it's not `js`. That's why. (facepalm)

**Comment by JimiC** [comment]
Should we add [YamlLint](https://github.com/adrienverge/yamllint)?

**Comment by robertohuertasm** [comment]
I don't see any problem with that.

**Comment by JimiC** [comment]
Unfortunately [yaml-lint](https://www.npmjs.com/package/yaml-lint) is at a very premature state and doesn't support rules. There is no point in adding a `yaml linter` atm.

**Pull request merged**